core/command/report.js

```diff
@@ -88,7 +125,13 @@ function writeBrowserReport (config, reporter) {
       throw err;
     });
 
-    return allSettled([jsonpConfgWrite, jsonConfgWrite]);
+    let promises = [jsonpConfgWrite, jsonConfgWrite];
+
+    if (config.archiveReport) {
+      promises.push(archiveReport(config));
```

Comment:
Hey, are `jsonpConfigWrite` and `jsonConfigWrite` guaranteed to have completed when we call `archiveReport()`?  I _think_ these are dependencies.

Reply:
Good point. Fixed.